scripts/odoo_revision_17.0.2024.0209

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+ODOO_REVISION=17.0.2024.0209
+
+# Checkout odoo repo
+cd odoo && git checkout 5956786c346b63c92a191b008812376cb88e9cb6
+
+# Create tag on the odoo repo
+git tag -a $ODOO_REVISION -m "$ODOO_REVISION"
+..
```

Comment:
The `..` are from zsh and not bash.

Rename `..` to `cd ..`

Use regex `^..` to make the replacement

Reply:
@laszlokr The regex is probably `^\.\.`